lib/tag-finder.coffee

```diff
@@ -8,14 +8,14 @@ SelfClosingTags = require './self-closing-tags'
 module.exports =
 class TagFinder
   constructor: (@editor) ->
-    @tagPattern = /(<(\/?))([^\s>]+)([\s>]|$)/
-    @wordRegex = /[^>\r\n]*/
+    @tagPattern = /(<(\/?))([^/\s>]+)([^/>]*)(\/?)(>)/
```

Comment:
I noticed that the first time around, but I copied the sequence again, and made sure to escape _all_ the slashes (I used a regex tester and it told me to escape even the `/`). That seems to have fixed the problem. Thanks for you help. 

I'll go ahead and push the changes and write a spec for the new cases.